clearing_operations/scripts/GRGTools.py

```diff
@@ -0,0 +1,348 @@
+
+    def getParameterInfo(self):
+        '''
+        Define parameter definitions
+        '''
+
+        input_area_features = arcpy.Parameter(name='input_canvas_area',
+                                              displayName='Input GRG Area',
+                                              direction='Input',
+                                              datatype='GPFeatureRecordSetLayer',
+                                              parameterType='Required',
+                                              enabled=True,
+                                              multiValue=False)
+        input_area_features.value = os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                                                 "layers", "GRGInputArea.lyr")
+
+        cell_width = arcpy.Parameter(name='cell_width',
+                                     displayName='Cell Width',
+                                     direction='Input',
+                                     datatype='GPDouble',
+                                     parameterType='Required',
+                                     enabled=True,
+                                     multiValue=False)
+        cell_width.value = 100.0
+
+        cell_height = arcpy.Parameter(name='cell_height',
+                                      displayName='Cell Height',
+                                      direction='Input',
+                                      datatype='GPDouble',
+                                      parameterType='Required',
+                                      enabled=True,
+                                      multiValue=False)
+        cell_height.value = 100.0
+
+        cell_units = arcpy.Parameter(name='cell_units',
+                                     displayName='Cell Units',
+                                     direction='Input',
+                                     datatype='GPString',
+                                     parameterType='Required',
+                                     enabled=True,
+                                     multiValue=False)
+        cell_units.filter.type = 'ValueList'
+        # cell_units.filter.list = ['METERS', 'FEET']
+        cell_units.filter.list = ['Meters', 'Feet']
+        cell_units.value = cell_units.filter.list[0]
+
+        label_start_position = arcpy.Parameter(name='label_start_position',
+                                               displayName='Label Start Position',
+                                               direction='Input',
+                                               datatype='GPString',
+                                               parameterType='Required',
+                                               enabled=True,
+                                               multiValue=False)
+        label_start_position.filter.type = 'ValueList'
+        # label_start_position.filter.list = ['UPPER_LEFT',
```

Comment:
Specifically for GRGTools.py in Parameters, we should be using UPPER_CASE list entities. But because we were only to do equivalency with previous tools I left them alone. This is also why there's a lot of other stuff that was commented out ... because it was in the previous tools.